contrib/core/actions/pause.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from random import randint
+from st2actions.runners.pythonrunner import Action
+
+
+class pause(Action):
+    def run(self, max_pause, random):
+        if random:
+            delay = randint(1, int(max_pause))
+        else:
+            delay = max_pause
+
+        time.sleep(delay)
+
+        return
```

Comment:
Add it to return the delay value.. makes no difference to me. but may be useful for some.